### ISSUE TITLE
Better display parallel transformer loading

### ISSUE THREAD

**Issue by DartBot** [dart-lang/pub#1000]
<a href="https://github.com/nex3"><img src="https://avatars.githubusercontent.com/u/188?v=3" align="left" width="96" height="96"hspace="10"></img></a> **Issue by [nex3](https://github.com/nex3)**
_Originally opened as dart-lang/sdk#19166_

----

Currently, pub loads all transformer plugins in sequence to ensure that dependencies between them are satisfied. However, there are cases where this isn't necessary. For example, suppose the application uses transformers from package A and package B, but those packages have no dependencies on one another. In that case, their transformers could be loaded in parallel, potentially leading to a considerable decrease in load times.

**Comment by DartBot** [comment]
<a href="https://github.com/nex3"><img src="https://avatars.githubusercontent.com/u/188?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [nex3](https://github.com/nex3)**

----

I was mistaken here: pub is already pretty good at transformer parallelization. What it's not good at is indicating that transformers are being loaded in parallel. Whenever a transformer is loaded, pub prints a progress indicator for that transformer, but if multiple transformers are loaded in parallel only the indicator for the last one survives. It would be nice to have this more accurately indicate to the user what's going on.
______
*Removed the owner.*
*Removed [Priority-High](../labels/Priority-High) label.*
*Added [Priority-Unassigned](../labels/Priority-Unassigned), [Triaged](../labels/Triaged) labels.*
*Changed the title to: "Better display parallel transformer loading".*

**Comment by DartBot** [comment]
<a href="https://github.com/nex3"><img src="https://avatars.githubusercontent.com/u/188?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [nex3](https://github.com/nex3)**

----

This was fixed a while ago.
______
*Added [Fixed](../labels/Fixed) label.*
